src/stat.h

```diff
@@ -69,9 +69,9 @@ class CStatBase
     virtual ~CStatBase(){};
     virtual UniValue GetNow() = 0; // Returns the current value of this statistic
     virtual UniValue GetTotal() = 0; // Returns the cumulative value of this statistic
-    virtual UniValue GetSeries(const std::string &name, int count) = 0; // Returns the historical or series data
+    virtual UniValue GetSeries(const std::string &namep, int count) = 0; // Returns the historical or series data
     // Returns the historical or series data along with timestamp
-    virtual UniValue GetSeriesTime(const std::string &name, int count) = 0;
+    virtual UniValue GetSeriesTime(const std::string &namep, int count) = 0;
 };
```

Comment:
why use &namep rather than &_name?   .. same question in a few other files.  If you mean it to be a pointer then the convention throughout bitcoin is to prefix with a 'p', so it would be pname (as we do for example with pnode), but in this case it's a reference so i would think &_name would be appropriate.

Reply:
With the ending p I meant parameter, but you are right that it is ambiguous. Going to use a _ prefix in all these cases